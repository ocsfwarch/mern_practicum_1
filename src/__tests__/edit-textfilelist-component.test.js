import React from 'react';
import EditTextFileList from '../components/edit-textfilelist.component.js';
import renderer from 'react-test-renderer';

it('Renders a snapshot', () => {
  const match = {
    params:{id:"1234"}
  };

  const tree = renderer.create(<EditTextFileList match={match}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
