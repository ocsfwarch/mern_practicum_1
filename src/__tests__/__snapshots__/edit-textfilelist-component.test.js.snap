// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a snapshot 1`] = `
<div>
  <h3>
    View Transcription
  </h3>
  <form>
    <div
      className="form-group"
    >
      <label>
        Name: 
      </label>
      <input
        className="form-control"
        readOnly="readOnly"
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Description: 
      </label>
      <input
        className="form-control"
        readOnly="readOnly"
        type="text"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Transcripton: 
      </label>
      <textarea
        className="form-control"
        readOnly="readOnly"
        rows="10"
        value=""
      />
    </div>
    <div
      className="form-group"
    >
      <input
        className="btn btn-primary"
        onClick={[Function]}
        type="button"
        value="Close"
      />
    </div>
  </form>
</div>
`;
